doc/tools/opensda.rst

```diff
@@ -59,169 +59,29 @@ application binary. :ref:`nxp_opensda_firmware` with this application.
-       CHK     misc/generated/configs.c
-       UPD     misc/generated/configs.c
-       CHK     include/generated/offsets.h
-       UPD     include/generated/offsets.h
-       CC      lib/libc/minimal/source/stdlib/strtol.o
-
-     <snip>
-
-       CC      kernel/work_q.o
-       AR      kernel/lib.a
-       CC      src/main.o
-       LD      src/built-in.o
-       AR      libzephyr.a
-       LINK    zephyr.lnk
-       IRQ     isr_tables.c
-       CC      isr_tables.o
-       LINK    zephyr.elf
-       BIN     zephyr.bin
-     Flashing frdm_k64f
-     Flashing Target Device
-     INFO:root:DAP SWD MODE initialised
-     INFO:root:K64F not in secure state
-     INFO:root:ROM table #0 @ 0xe00ff000 cidr=b105100d pidr=4000bb4c4
-     INFO:root:[0]<e000e000:SCS-M4 cidr=b105e00d, pidr=4000bb00c, class=14>
-     WARNING:root:Invalid coresight component, cidr=0x0
-     INFO:root:[1]<e0001000: cidr=0, pidr=0, component invalid>
-     INFO:root:[2]<e0002000:FPB cidr=b105e00d, pidr=4002bb003, class=14>
-     WARNING:root:Invalid coresight component, cidr=0x1010101
-     INFO:root:[3]<e0000000: cidr=1010101, pidr=101010101010101, component invalid>
-     WARNING:root:Invalid coresight component, cidr=0x0
-     INFO:root:[4]<e0040000: cidr=0, pidr=0, component invalid>
-     INFO:root:[5]<e0041000:ETM-M4 cidr=b105900d, pidr=4000bb925, class=9, devtype=13, devid=0>
-     INFO:root:[6]<e0042000:ETB cidr=b105900d, pidr=4003bb907, class=9, devtype=21, devid=0>
-     INFO:root:[7]<e0043000:CSTF cidr=b105900d, pidr=4001bb908, class=9, devtype=12, devid=28>
-     INFO:root:CPU core is Cortex-M4
-     INFO:root:FPU present
-     INFO:root:6 hardware breakpoints, 4 literal comparators
-     INFO:root:4 hardware watchpoints
-     [====================] 100%
-     INFO:root:Programmed 12288 bytes (3 pages) at 10.57 kB/s
-     make[2]: Leaving directory '/home/maureen/zephyr/samples/hello_world/outdir/frdm_k64f'
-     make[1]: Leaving directory '/home/maureen/zephyr'
+Use the CMake ``flash`` target with the argument ``OPENSDA_FW=daplink`` to
+build your Zephyr application, invoke the pyOCD flash tool and program your
+Zephyr application to flash. Some boards set the argument by default, so it is
+not always necessary to set explicitly.
 
+  .. zephyr-app-commands::
+     :zephyr-app: samples/hello_world
+     :gen-args: -DOPENSDA_FW=daplink
+     :goals: flash
 
 Debugging
 =========
 
-Use the ``make debug`` build target with ``OPENSDA_FW=daplink`` to build your
-Zephyr application, invoke the pyOCD GDB server, attach a GDB client, and
-program your Zephyr application to flash. It will leave you at a gdb prompt.
-
-  .. code-block:: console
+Use the CMake ``debug`` target with the argument ``OPENSDA_FW=daplink`` to
+build your Zephyr application, invoke the pyOCD GDB server, attach a GDB
+client, and program your Zephyr application to flash. It will leave you at a
+GDB prompt. Some boards set the argument by default, so it is not always
```

Comment:
you used a lowercase "gdb" in your change to mimxrt1050_evk.rst  (pick one)

Reply:
Fixed mimxrt1050_evk.rst to use uppercase